lava_test_plans/projects/lkft-android/boot-empty.yaml

```diff
@@ -0,0 +1 @@
+../../testcases/boot.yaml
```

Comment:
why you want to run like the following:
```
python3 ./submit_for_testing.py
--device-type x15
--variables android-variables.ini
--test-case boot.yaml
--dry-run
````
from what I understand, this command is to use the generic x15 device settings and the generic boot testcase to generate the lava definition, but with android variables.
any meaning to generate the generic definition with android variables?

Reply:
with the following command, if there is no boot-empty.yaml link file, I guess it will use the boot.yaml under lava_test_plans/projects/lkft-android, which is the link of ../../testcases/android-boot.yaml.
```
python3 ./submit_for_testing.py
--device-type x15
--variables ./lava_test_plans/projects/lkft-android/variables.ini
--testplan-path ./lava_test_plans/projects/lkft-android/
--testplan-device-path ./lava_test_plans/projects/lkft-android/devices/
--test-case boot.yaml
--dry-run
```

Reply:
the link of the link of ../../testcases/android-boot.yaml. will be added later

Reply:
hmm, I think you are right, with this change, the lava_test_plans/projects/lkft-android/boot-empty.yaml  link could be removed, the testcases/boot.yaml will be used, and after we added the link to ../../testcases/android-boot.yaml,
if necessary we could add the boot-empty.yaml link again.